### PR TITLE
add product card

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_product-cards/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.container__card {
+  padding: 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/style.css:22]: move `box-sizing` to `*` selector

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/style.css:23]: Move `font-family` to `html` selector

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  width: 200px;
+  border: 1px solid transparent;
```
Comment (IvanFesenko) [inline src/style.css:25]: Check border color from design

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
```
Comment (IvanFesenko) [inline src/style.css:46]:
```suggestion
  line-height: 1.4;
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  height: 134px;
+}
+
+.card__title {
+  margin: 0;
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: #060b35;
```
Comment (IvanFesenko) [inline src/style.css:40]: Move to variable and use in all places

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check design, card should have border.
Stars should work with any modifier (1-5).
Check my comments

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+body,
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  
+  --dark-grey: #060b35;
+  --light-grey: #616070;
+  --blue: #00acdc;
+  --white: #fff;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:14]: initialize a variables in :root selector on next times

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 24px;
+}
+
+.card__price-text {
+  font-weight: 400;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:101]: redundant property, weight is already 400

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+.review__stars--1 :nth-child(1),
+.review__stars--2 :nth-child(-n+2),
+.review__stars--3 :nth-child(-n+3),
+.review__stars--4 :nth-child(-n+4),
+.review__stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+.review-reviews {
+  font-weight: 400;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:88]: same

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Great job, almost there!

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,57 @@
...
+          alt="APPLE A1419 iMac 27"
+        >
+    
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+    
+        <p class="card__code">Product code: 195434</p>
+    
+        <div class="card__review review">
+          <div class="review__stars review__stars--4">
+            <div class="review__stars-star"></div>
```
Comment (polosanya) [inline src/index.html:32]:
```suggestion
        <div class="card__review ">
          <div class="card__stars stars stars--4">
            <div class="stars__star"></div>
```
You don't need to make the review a separate block. There should be only 2 block in this task
image

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+a {
+  text-decoration: none;
+}
+
+.container__card {
+  padding: 16px;
+}
+
+.card {
```
Comment (polosanya) [inline src/style.css:31]: Create separate files for styling `card` and `stars` blocks
image

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+  text-align: center;
+  text-transform: uppercase;
+  background: var(--blue);
+  border-radius: 5px;
+}
+
+.card__buy:hover {
+  color: var(--blue);
+  background: var(--white);
+  border: 1px solid var(--blue);
```
Comment (polosanya) [inline src/style.css:130]: image

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+.card__buy {
+  display: block;
+  font-weight: 700;
+  font-size: 14px;
+  color: var(--white);
+  margin-top: 16px;
+  padding: 11px 70px;
+  text-align: center;
+  text-transform: uppercase;
+  background: var(--blue);
```
Comment (polosanya) [inline src/style.css:123]:
```suggestion
  background-color: var(--blue);
```

**Review by polosanya — CHANGES_REQUESTED** [review]
A few small but important fixes

**Review by polosanya — CHANGES_REQUESTED** [review]
Please add all new files to your commit
image
image

**Pull request closed without merging**